cloudtrail/lib/botocore/data/fsx/2018-03-01/examples-1.json

```diff
@@ -0,0 +1,438 @@
+        },
+        "output": {
+          "Backup": {
+            "BackupId": "backup-0a3364eded1014b28",
+            "CreationTime": 1617954808.068,
+            "FileSystem": {
+              "FileSystemId": "fs-0498eed5fe91001ec",
+              "FileSystemType": "LUSTRE",
+              "LustreConfiguration": {
+                "AutomaticBackupRetentionDays": 0,
+                "DeploymentType": "PERSISTENT_1",
+                "PerUnitStorageThroughput": 50,
+                "WeeklyMaintenanceStartTime": "1:05:00"
+              },
+              "ResourceARN": "arn:aws:fsx:us-east-1:012345678912:file-system/fs-0f5179e395f597e66",
+              "StorageCapacity": 2400,
+              "StorageType": "SSD"
+            },
+            "KmsKeyId": "arn:aws:fsx:us-east-1:012345678912:key/d1234e22-543a-12b7-a98f-e12c2b54001a",
+            "Lifecycle": "COPYING",
+            "OwnerId": "123456789012",
+            "ResourceARN": "arn:aws:fsx:us-east-1:012345678912:backup/backup-0a3364eded1014b28",
+            "Tags": [
+              {
+                "Key": "Name",
+                "Value": "MyBackup"
+              }
+            ],
+            "Type": "USER_INITIATED"
+          }
+        },
+        "comments": {
+        },
+        "description": "This operation copies an Amazon FSx backup.",
+        "id": "to-copy-a-backup-1481847318640",
+        "title": "To copy a backup"
+      }
+    ],
+    "CreateBackup": [
+      {
+        "input": {
+          "FileSystemId": "fs-0498eed5fe91001ec",
+          "Tags": [
+            {
+              "Key": "Name",
+              "Value": "MyBackup"
+            }
+          ]
+        },
+        "output": {
+          "Backup": {
+            "BackupId": "backup-03e3c82e0183b7b6b",
+            "CreationTime": "1481841524.0",
+            "FileSystem": {
+              "FileSystemId": "fs-0498eed5fe91001ec",
+              "OwnerId": "012345678912",
+              "StorageCapacity": 300,
+              "WindowsConfiguration": {
+                "ActiveDirectoryId": "d-1234abcd12",
+                "AutomaticBackupRetentionDays": 30,
+                "DailyAutomaticBackupStartTime": "05:00",
+                "WeeklyMaintenanceStartTime": "1:05:00"
+              }
+            },
+            "Lifecycle": "CREATING",
+            "ProgressPercent": 0,
+            "ResourceARN": "arn:aws:fsx:us-east-1:012345678912:backup/backup-03e3c82e0183b7b6b",
+            "Tags": [
+              {
+                "Key": "Name",
+                "Value": "MyBackup"
+              }
+            ],
+            "Type": "USER_INITIATED"
+          }
+        },
+        "comments": {
+        },
+        "description": "This operation creates a new backup.",
+        "id": "to-create-a-new-backup-1481840798597",
+        "title": "To create a new backup"
+      }
+    ],
+    "CreateFileSystem": [
+      {
+        "input": {
+          "ClientRequestToken": "a8ca07e4-61ec-4399-99f4-19853801bcd5",
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: Critical

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

 <!-- 80be254d-d180-403e-9f93-83feb40a089d -->